docs/tools/cli/mirrormaker.rst

```diff
@@ -0,0 +1,221 @@
+  * - ``--target-cluster``
+    - The Aiven for Apache Kafka service name to be used as target for replication
+  * - ``replication_flow_config``
+    - JSON string or path (preceded by ``@``) to a JSON configuration file for the replication flow definition
+
+**Example:** In the service ``kafka-mm`` create a new replication flow from an Aiven for Apache Kafka service with integration alias ``kafka-source-alias`` to a service named ``kafka-target-alias`` with the following settings:
+
+* include all topics with name starting with ``my-src-topic`` (topic name patterns can be defined using `Java patterns <https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html>`_)
+* exclude all topics with name ending with ``not-include``
+* ``DefaultReplicationPolicy`` as replication policy class
+* enable MirrorMaker2 heartbeats
+* enable synching of consumer groups offset every ``60`` seconds
+
+::
+
+  avn mirrormaker replication-flow create kafka-mm \
+    --source-cluster kafka-source-alias \
+    --target-cluster kafka-target-alias \
+    '
+      {
+          "emit_heartbeats_enabled": true,
+          "enabled": true,
+          "replication_policy_class": "org.apache.kafka.connect.mirror.DefaultReplicationPolicy",
+          "source_cluster": "kafka-source-alias",
+          "sync_group_offsets_enabled": true,
+          "sync_group_offsets_interval_seconds": 60,
+          "target_cluster": "kafka-target-alias",
+          "topics": [
+              "my-src-topic.*"
+          ],
+          "topics.blacklist": [
+              ".*not-include"
+          ]
+      }
+    '
+
+``avn mirrormaker replication-flow delete``
+'''''''''''''''''''''''''''''''''''''''''''
+
+Deletes an existing Aiven for Apache Kafka MirrorMaker2 replication flow. 
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The Aiven for Apache Kafka MirrorMaker2 service name where to create the replication flow
+  * - ``--source-cluster``
+    - The Aiven for Apache Kafka service name to be used as source for replication
+  * - ``--target-cluster``
+    - The Aiven for Apache Kafka service name to be used as target for replication
+
+
+**Example:** In the service ``kafka-mm`` delete the replication flow from an Aiven for Apache Kafka service with integration alias ``kafka-source-alias`` to the service named ``kafka-target-alias``.
+
+::
+
+  avn mirrormaker replication-flow delete kafka-mm \
+    --source-cluster kafka-source-alias \
+    --target-cluster kafka-target-alias
+
+``avn mirrormaker replication-flow get``
+'''''''''''''''''''''''''''''''''''''''''''
+
+Retrieves the configuration details of an existing Aiven for Apache Kafka MirrorMaker2 replication flow. 
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The Aiven for Apache Kafka MirrorMaker2 service name where to create the replication flow
```

Comment:
We're not creating, but getting - so "where to get a replication flow"

Reply:
fixed